rbac/management/audit_log/model.py

```diff
@@ -98,6 +97,21 @@ def find_edited_field(self, resource, resource_name, request, object):
                 description = description + "edited access (permissions/resources)"
         return description
 
+    def find_specific_list_of_users(self, type_dict, user_type):
+        """Create list of principals/roles/service accounts for description."""
+        names_list = []
+        if user_type == AuditLog.USER:
+            for i in type_dict:
+                names_list.append(i["username"])
+        elif user_type == "service_account":
+            for i in type_dict:
+                names_list.append(i["clientId"])
+        elif user_type == AuditLog.ROLE:
+            names_list = type_dict
+        else:
+            return NameError("User type does not exist")
```

Comment:
I would prefer here the ValueError because the NameError is more suitable for scenarios when you try to use/call some variable that is not defined